wdl/qc_imputation.wdl

```diff
@@ -375,14 +783,14 @@ task batch_qc {
         $plink2_cmd --bfile ${base} --remove <(awk '{OFS="\t"; print "0",$1}' ${base}.samples_exclude.txt | sort -u) \
```

Comment:
not related to new changes but just occurred that we are checking that the sex has to match both the sex in input fams as well as ssn checks. I think we discussed that it is fine as the fams are delivered with the assumed sex correctly always.... we could make this more general by comparing only to the provided ssn list. Fine for this though but just a thought to simplify and minimize potential side effects if using in slightly different scenario.

Reply:
I updated an issue to get this stint to an end fast, call caching and all that

Reply:
yep great!!